.nuget/NuGet.targets

```diff
@@ -20,8 +20,8 @@
         <!-- Package sources used to restore packages. By default, registered sources under %APPDATA%\NuGet\NuGet.Config will be used -->
         <!-- The official NuGet package source (https://nuget.org/api/v2/) will be excluded if package sources are specified and it does not appear in the list -->
         
-            <PackageSource Include="https://nuget.org/api/v2/" />
-            <PackageSource Include="http://nupeek.internal.azavea.com" />
+            <PackageSource Include="http://nuget.org/api/v2/" />
```

Comment:
Not a big deal, since this is read only access to public packages, but why do we need to drop https?